### ISSUE TITLE
Why don't we update the documentation for the `pre-commit integration` section?

### ISSUE THREAD

**Issue by officel** [rvben/rumdl#666]
### rumdl version

0.2.14

### Installation method

Other

### Bug description

I wasn't sure if I should submit a pull request, so 🙇 

now on readme.md

```yaml
repos:
  - repo: https://github.com/rvben/rumdl-pre-commit
    rev: v0.2.14
    hooks:
      - id: rumdl      # Lint only (fails on issues)
      - id: rumdl-fmt  # Auto-format and fail if issues remain
```

see https://github.com/rvben/rumdl-pre-commit/blob/v0.2.14/.pre-commit-hooks.yaml#L4

from https://github.com/rvben/rumdl-pre-commit/blob/v0.1.64/.pre-commit-hooks.yaml

maybe

```yaml
repos:
  - repo: https://github.com/rvben/rumdl-pre-commit
    rev: v0.2.14
    hooks:
      - id: rumdl      # Lint and auto-fix Markdown files (fails on issues)
      - id: rumdl-fmt  # Auto-format and fail if issues remain
```

### Steps to reproduce

```shell
$ prek run -a
```

I was surprised that the fix ran automatically when I ran the command.

### Expected behavior

It would be nice if users could decide for themselves whether they want auto-correction enabled or disabled.

### Actual behavior

Since the document hasn't been updated, I'm surprised that it gets automatically corrected without any warning.

If auto-correction isn't needed, does this mean I have to write it like this?

```yaml
  - repo: https://github.com/rvben/rumdl-pre-commit
    rev: v0.2.14
    hooks:
      - id: rumdl      # Lint only (fails on issues)
        entry: rumdl check
```

### Configuration

```toml

```

### Operating system

Linux

### Additional context

I think the best approach would be to use the `args` in `pre-commit` to pass the `--fix` option.
Rewriting the `entry` feels a bit clunky.
For now, I think it would be helpful to align it with the documentation. 🙇

**Comment by rvben** [comment]
You're right, and thanks for the nudge - digging in, the docs weren't the only thing off; the default itself was backwards.

I checked what the rest of the ecosystem does, and the bare linter hook is non-destructive everywhere:

- **ruff**: `ruff` / `ruff-check` runs `ruff check` (no fix); you opt in with `args: [--fix]`
- **markdownlint-cli**: `markdownlint` lints; a separate `markdownlint-fix` hook does the fixing

rumdl's `rumdl check --fix` default was the odd one out - exactly the surprise you hit. So rather than just fix the wording, I flipped the default to match: the `rumdl` hook now runs `rumdl check` (lint-only, never rewrites your files), and auto-fix is an explicit opt-in via `args` - the approach you suggested:

```yaml
  - repo: https://github.com/rvben/rumdl-pre-commit
    rev: v0.2.15
    hooks:
      - id: rumdl
        args: [--fix]   # opt in to auto-fix
      - id: rumdl-fmt
```

`rumdl-fmt` is unchanged. This shipped in **v0.2.15** (just released), and the README/docs are updated to match. If you relied on the old auto-fixing behavior, add `args: [--fix]` to restore it.

Thanks again for flagging this so clearly.

**Comment by officel** [comment]
Thank you for checking this.
I think this is a great solution.
I’ve confirmed that it works!

**Comment by rvben** [comment]
Fixed in v0.2.15 - the `rumdl` hook is now lint-only by default, with auto-fix available via `args: [--fix]`. Thanks again for the report!
